### DISCUSSION TITLE
Bluetooth connection failure

### DISCUSSION THREAD

**Discussion by quadeqx** [dat-linux/community#50] [Issues]
I've tried to connect my Bluetooth device using dat Linux but this is what it displays. My Bluetooth device is on and discoverable. How can I resolve the issue?
![BT ERROR](https://github.com/dat-linux/community/assets/154374993/885ec2c7-e4e4-4a07-88c4-b24914b7ad55)

**Comment by dat-linux** [comment]
Hi, try installing "Bluetooth Manager" and see if that helps.

**Comment by quadeqx** [comment]
I've tried installing bluez but it keeps telling me unable to locate
package build-essentials. What should I do?

**Reply by dat-linux** [reply]
`sudo apt-get install build-essential`

**Comment by quadeqx** [comment]
this is a display after trying to reinstall bluez. Any other suggestion

**Comment by quadeqx** [comment]
![Uploading blt_scrn.png…]()
